argocd/secret/argo-cd-https-secret.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: argocd-private-repo
+  namespace: argocd
+  labels:
+    argocd.argoproj.io/secret-type: repository
+stringData:
+  type: git
+  url: https://github.com/andrii-yeremenko/news-aggregator.git
+  username: ArgoCd
+  password: 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
```

Comment:
??

Reply:
this secret has readOnly permission; since our repository is public, there is no need to hide it.